qchem/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 <h3>Improvements</h3>
 
+* Updates `PennyLane-QChem` functions to separately take as input the
```

Comment:
Suggest removing mention of `PennyLane-QChem`, and instead listing the functions that have changed with bullet points :)

Reply:
The changelog is a good place to provide all the fine-grained detail. Imagine a user reading *just* the changelog - you'll want them to be able to walk away knowing what changes they'll need to make to their code.